### PR TITLE
Enable backup and restore operations for KeyVault secrets

### PR DESCRIPTION
**Pull request by dragav**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as a link to the swagger spec, used to generate the code.
- [x] The `project.json` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Review by pomortaz — APPROVED** [review]

**Comment by shahabhijeet** [comment]
@azuresdkci add to whitelist

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist.

File: src/KeyVault/Microsoft.Azure.KeyVault/Generated/KeyVaultClientExtensions.cs [hunk 1]
```diff
@@ -845,23 +845,69 @@ public static partial class KeyVaultClientExtensions
...
+        /// <param name='vaultBaseUrl'>
+        /// The vault name, for example https://myvault.vault.azure.net.
+        /// </param>
+        /// <param name='secretBundleBackup'>
+        /// The backup blob associated with a secret bundle.
+        /// </param>
+        /// <param name='cancellationToken'>
+        /// The cancellation token.
+        /// </param>
+        public static async Task<SecretBundle> RestoreSecretAsync( this IKeyVaultClient operations, string vaultBaseUrl, byte[ ] secretBundleBackup, CancellationToken cancellationToken = default( CancellationToken ) )
```
Comment (shahabhijeet) [inline src/KeyVault/Microsoft.Azure.KeyVault/Generated/KeyVaultClientExtensions.cs]: @dragav can you bump up the version of the nugget.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/KeyVault/Microsoft.Azure.KeyVault/Generated/KeyVaultClientExtensions.cs (hunk 1 above)
Comment (dragav) [inline src/KeyVault/Microsoft.Azure.KeyVault/Generated/KeyVaultClientExtensions.cs]: @shahabhijeet That was already done by this [PR](https://github.com/Azure/azure-sdk-for-net/pull/3021), and we'd like to stay on this version until ready to GA. A few other small features will be brought into the same SDK ver (2.1.0-preview).

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**